kgforge/specializations/models/rdf/store_service.py

```diff
@@ -28,37 +31,43 @@
 
 class StoreService(RdfService):
 
-    def __init__(self, default_store: Store, context_iri: Optional[str] = None,
-                 context_store: Optional[Store] = None) -> None:
+    def __init__(
+        self,
+        default_store: Store,
+        context_iri: Optional[str] = None,
+        context_store: Optional[Store] = None,
+    ) -> None:
 
         self.default_store = default_store
         self.context_store = context_store or default_store
         # FIXME: define a store independent strategy
-        self.NXV = Namespace(self.default_store.service.namespace) if hasattr(self.default_store.service, "namespace") \
-            else Namespace(Service.NEXUS_NAMESPACE_FALLBACK)
-        self.store_metadata_iri = self.default_store.service.store_context if hasattr(self.default_store.service, "store_context") \
+        self.store_metadata_iri = (
+            self.default_store.service.store_context
+            if hasattr(self.default_store.service, "store_context")
             else Namespace(Service.NEXUS_CONTEXT_FALLBACK)
-        self._shapes_to_resources: Dict
-        self._imported = []
-        self._graph = Graph()
+        )
+        self._graph = rdflib.Dataset()
```

Comment:
Similarly to my comment on the directory service, the rdflib.Dataset() instance should be passed to the super's constructor so that it sets self._graph.
With respect to self._sg needing the dataset instance, I would suggest that self._sg be initialized in the superclass, also because in RdfService code, there is an assumption made that self._sg exists, even though it isn't initialized in its constructor (all its subclasses initialise it but this isn't safe)